src/lang/fr.json

```diff
@@ -0,0 +1,314 @@
+  "FSC.Configuration.Moon.CycleLengthHelp": "La longueur du cycle lunaire en jours. Accepte les décimales.",
+  "FSC.Configuration.Moon.CycleAdjustment": "Réglage du cycle",
+  "FSC.Configuration.Moon.CycleAdjustmentHelp": "Le nombre de jours pour ajuster le calcul lors de la détermination de la phase actuelle.",
+  "FSC.Configuration.Moon.Color": "Couleur de la lune",
+  "FSC.Configuration.Moon.ColorHelp": "La couleur à associer à cette lune.",
+  "FSC.Configuration.Moon.Add": "Ajouter une nouvelle lune",
+  "FSC.Configuration.Moon.RemoveAll": "Supprimer toutes les lunes",
+  "FSC.Configuration.Moon.Phases": "Phases",
+  "FSC.Configuration.Moon.PhaseName": "Nom de phase",
+  "FSC.Configuration.Moon.PhaseNameHelp": "Le nom de cette phase de la lune.",
+  "FSC.Configuration.Moon.PhaseLength": "Longueur de phase",
+  "FSC.Configuration.Moon.PhaseLengthHelp": "Combien de temps dure la phase en jours.",
+  "FSC.Configuration.Moon.PhaseSingleDay": "Phase 1 Journée",
+  "FSC.Configuration.Moon.PhaseSingleDayHelp": "Si cette phase de lune ne doit se produire que sur 1 jour.",
+  "FSC.Configuration.Moon.PhaseIcon": "Icône de phase",
+  "FSC.Configuration.Moon.PhaseIconHelp": "L'icône à associer à cette phase de lune.",
+  "FSC.Configuration.Moon.PhaseAdd": "Ajouter une nouvelle phase de lune",
+  "FSC.Configuration.Moon.PhaseRemoveAll": "Supprimer toutes les phases de la lune",
+  "FSC.Configuration.Moon.FirstNewMoon": "Référence Nouvelle Lune",
+  "FSC.Configuration.Moon.YearReset": "Réinitialiser l'année de la lune de référence",
+  "FSC.Configuration.Moon.YearResetHelp": "Si l'année de la lune de référence doit être réinitialisée et si c'est le cas quand.",
+  "FSC.Configuration.Moon.YearResetNo": "Ne pas réinitialiser l'année de référence",
+  "FSC.Configuration.Moon.YearResetLeap": "Réinitialiser l'année de référence chaque année bissextile",
+  "FSC.Configuration.Moon.YearResetX": "Réinitialiser l'année de référence tous les X ans",
+  "FSC.Configuration.Moon.YearX": "Réinitialiser les années lunaires de référence",
+  "FSC.Configuration.Moon.YearXHelp": "Réinitialiser l'année de référence des nouvelles lunes tous les x nombre d'années.",
+  "FSC.Configuration.Moon.Year": "Année de la Nouvelle Lune",
+  "FSC.Configuration.Moon.YearHelp": "L'année d'une nouvelle lune à utiliser comme référence.",
+  "FSC.Configuration.Moon.Month": "Mois de la Nouvelle Lune",
+  "FSC.Configuration.Moon.MonthHelp": "Le mois d'une nouvelle lune à utiliser comme référence.",
+  "FSC.Configuration.Moon.Day": "Jour de la Nouvelle Lune",
+  "FSC.Configuration.Moon.DayHelp": "Le jour d'une nouvelle lune à utiliser comme référence.",
+  "FSC.Configuration.Moons": "Lunes",
+  "FSC.Warn.Time.ActiveCombats": "Il y a un ou des combats actifs en cours, veuillez les résoudre avant de démarrer l'horloge en temps réel.",
+  "FSC.Warn.Notes.NotGM": "Il n'y a pas de MJ présent dans le jeu, un MJ doit être connecté pour que les joueurs puissent ajouter des notes.",
```

Comment:
I would replace "MJ" by "Maitre de jeu"

Reply:
But in the english file it's **GM**, not the full name. Only the abbreviation. MJ is usual abbreviation in french. If with expand the two letters, maybe you could use "Meneur de Jeu". (to follow the trend of removing the notion of master/slave)

Reply:
I've never heard someone use "MJ" in a TTRPG context (everyone in my groups says DM, so... yeah, when talking we don't translate). So when I saw it I had to go back in the original string to see what it meant. This is the only problem I have with it.

As to whether we should follow the trend of removing master/slave, I'm just going to say that we should stick to the most used term. I wont go further in that debate (because this is GitHub, not a socio-political forum and it makes me uncomfortable)